content/cfp.html

```diff
@@ -0,0 +1,82 @@
+
+<p>Note that there is a <a href="#financial-support">programme of financial support</a> for speakers.</p>
+
+<h3>We <em>especially</em> want proposals from...</h3>
+
+<em>People who are minorities in this field</em>
+<p><a href="/diversity-accessibility">Diversity at PyCon</a> is important to us.</p>
+
+<p>If you can help us make this event more diverse just by being there, please, do come forward with a proposal. We need you.</p>
+
+<em>First-time speakers</em>
+<p>If you haven’t spoken at a conference before, this is your invitation to do so. After all, we want to be remembered as the conference where you started your superstar speaking career. You’ll find that a Python/Django audience is friendly and receptive, so don’t be shy.</p>
+
+<strong>If you’d like any help in proposing, preparing or presenting your talk, whoever you are and whatever kind of help you’d find useful: please see our <a href="/speaker-mentors">speaker mentor programme</a>.</strong>
+
+<h3>Propose a...</h3>
+
+<h4>Talk</h4>
+<p>Do you have something to share with the rest of the community?</p>
+
+<p>Proposals on any subject are welcomed, at any level of technical expertise.</p>
+
+<p>Tell us about your experiments, discoveries, solutions, achievements and other adventures, using the talk proposal form.</p>
+
+<h5>Lightning talks</h5>
+
+<p>There’ll also be opportunities for five-minute lightning talks. You don’t need to register in advance; just put your name down on the day.</p>
+
+<h4>Workshop</h4>
+<p>Can you lead a tutorial, on a Python or Django topic?</p>
+
+<p>We’ll be holding introductory workshops on our open day, and more advanced sessions alongside the sprints on Thursday and Friday.</p>
+
+<p>Please don’t hesitate to suggest a workshop for the event, using the workshop proposal form. We’re open to all kinds of ideas, and especially ones we haven’t even thought of.</p>
+
+<h4>Code clinic</h4>
+<p>Could you be a friendly neighbourhood code doctor?</p>
+
+<p>Alongside the sprints, we’re holding code clinics, where an expert will be on hand to help with people’s Python/Django questions and problems.</p>
+
+<p>A clinic might run for an hour or two. If you like to offer a clinic on a particular subject, or even a general one, please use the workshop proposal form.</p>
+
+<h3>How to submit a proposal</h3>
+<p>There’s a proposal form for talks and a proposal form for tutorials and demonstrations. Take a look at them - they’ll make it clearer how this works.</p>
+
+<p>Don’t agonise over your proposal. Just get it in. We’ll get back to you if anything’s not clear.</p>
+
+<p>Don’t hesitate to submit more than one proposal.</p>
+
+<p>The deadline for submissions is the end of 10th May (UTC).</p>
```

Comment:
I arrived at this by working backwards from the conference date.  If we give people three months to prepare a talk, and spend a month choosing talks, we get the middle of May.

Is three months more time than's required?